sycl/source/detail/device_info.hpp

```diff
@@ -1155,6 +1155,13 @@ inline bool get_device_info_host<info::device::kernel_kernel_pipe_support>() {
   return false;
 }
 
+template <>
+inline std::string get_device_info_host<info::device::backend_version>() {
+  throw runtime_error(
+      "Backend version feature is not supported on HOST device.",
+      PI_INVALID_DEVICE);
+}
+
```

Comment:
Do I understand correctly that it's only a stub and going to be modified in follow-up patch?

Reply:
Yes it is a stub, but we currently do not have plans for implementing it for other backends. I am not even sure what it should return for host, as it is supposed to be defined by backend specification.

Reply:
It could just throw an exception for host.